aait/backend/group-2A/WebApi/Controllers/LikeController.cs

```diff
@@ -0,0 +1,71 @@
+using Application.DTO.CommentDTO;
+using Application.DTO.Like;
+using Application.DTO.Post;
+using Application.DTO.UserDTO;
+using Application.Features.Comment.Requests.Commands;
+using Application.Features.Comment.Requests.Queries;
+using Application.Features.Like.Request.Commands;
+using Application.Features.Like.Request.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebApi.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class LikeController : ControllerBase
+{
+    private readonly IMediator _mediator;
+
+    public LikeController(IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+    
+  
+    [HttpPost("{PostId}")]
+    public async Task<ActionResult> AddLike( int PostId)
```

Comment:
add [frombody] attribute to bind the postId from request body to postId paramater